api/src/models/bcmi/permit-bcmi.js

```diff
@@ -1,25 +1,46 @@
 module.exports = require('../../utils/model-schema-generator')(
   'PermitBCMI',
   {
+    // NRPTI Record boilerplate
     _schemaName: { type: String, default: 'PermitBCMI', index: true },
     _sourceRefId:  { type: String, default: null, index: true },
     _epicProjectId: { type: 'ObjectId', default: null, index: true },
-
     read: [{ type: String, trim: true, default: 'sysadmin' }],
     write: [{ type: String, trim: true, default: 'sysadmin' }],
-
+    recordName: { type: String, default: '' },
```

Comment:
Should any of these new fields be included on the master model? I am not sure if they should or not, but normally I think the models align fairly closely.

Reply:
Yeah, I wasn't sure so I figured it would come up in the review. Easy to match them up though so I can add them for master/flavour

Reply:
I've added the attributes to the Permit master and the LNG flavour. Noticed the PermitBCMI model was missing in common as well so I added that in for completeness.